src/web/web-service/frontend/index.html

```diff
@@ -199,27 +201,31 @@ <h4 style="margin: 18px 12px 12px 24px">OT Border Router</h4>
               <div layout="row">
                 <md-input-container flex="50">
                   <label>Network Key</label>
-                  <input required name="networkKey" ng-model="thread.networkKey">
+                  <input required minlength="32" maxlength="32" ng-pattern="/^[0-9a-fA-F]{32}$/" name="networkKey" ng-model="thread.networkKey">
                   <div ng-messages="threadForm.networkKey.$error">
-                    <div ng-message="required">This is required.</div>
+                    <div ng-message-exp="['required', 'minlength', 'maxlength', 'pattern']">
+                      Network Key must be 32 hexadecimal digits long.
+                    </div>
                   </div>
                 </md-input-container>
 
                 <md-input-container flex="50">
                   <label>Channel</label>
-                  <input required name="channel" ng-model="thread.channel">
+                  <input required name="channel" type="number" min="11" max="26" ng-model="thread.channel">
                   <div ng-messages="threadForm.channel.$error">
-                    <div ng-message="required">This is required.</div>
+                    <div ng-message-exp="['required', 'min', 'max']">Channel must be in 11~26.</div>
                   </div>
                 </md-input-container>
               </div>
 
               <div layout="row">
                 <md-input-container flex="100">
                   <label>On-Mesh Prefix</label>
-                  <input required name="prefix" ng-model="thread.prefix">
+                  <input required name="prefix" ng-model="thread.prefix" ng-pattern="/^((?!(fe80:)|(ff[0-9a-fA-F][0-9a-fA-F]:))([0-9a-fA-F]{1,4}:)(([0-9a-fA-F]{1,4}:){1,7}):((/[1-9][0-9]?){0,1}))$/">
```

Comment:
It seems this regex cannot work. Have you tried the following IPs?
```
FF01:0:0:0::/64
FE80:200:5aee::/64
fd11:22::/120
```
I guess we can add regex modifier to perform case-insensitive matching. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions